app/src/main/java/com/heapixLearn/discovery/ui/contact/ContactListAdapter.java

```diff
@@ -0,0 +1,130 @@
+package com.heapixLearn.discovery.ui.contact;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.bumptech.glide.Glide;
+import com.heapixLearn.discovery.R;
+
+import java.util.ArrayList;
+
+import de.hdodenhof.circleimageview.CircleImageView;
+
+public class ContactListAdapter extends RecyclerView.Adapter<ContactListAdapter.ViewHolder> {
+
+    private ArrayList<Contact> list = new ArrayList<>();
+    private Context context;
+    //TODO init manager
+    private ContactManager contactManager;
```

Comment:
@ViachaslauPratasenia I didn't found a place where you assign a value to the `contactManager` field